modules/electrophysiology_browser/php/electrophysiologybrowserrowprovisioner.class.inc

```diff
@@ -47,10 +47,17 @@ class ElectrophysiologyBrowserRowProvisioner
               s.Visit_label                             AS Visit_Label,
               MIN(pf.AcquisitionTime)                   AS Acquisition_Time,
               MIN(pf.InsertTime)                        AS Insert_Time,
-              GROUP_CONCAT(DISTINCT pot.OutputTypeName) AS Links,
+              GROUP_CONCAT(DISTINCT CASE 
+                WHEN pot.OutputTypeName='derivatives' THEN 'raw,derivative'
```

Comment:
Should be `WHEN pot.OutputTypeName='derivatives' THEN 'raw,derivatives'` otherwise the type is unrecognized ([sessions.inc.class](https://github.com/aces/Loris/blob/14677bfb3aa2b973c6cfc0d11737d3a9a8aa08cf/modules/electrophysiology_browser/php/sessions.class.inc#L134) only accept raw and derivatives as valid types).

Reply:
Christine requested that I change all "derivatives" to singular. Is it alright if I change sessions.class.inc to take "raw" or "derivative" as valid output types instead to match?

Reply:
I tend to think that renaming the fields in the query is prone to mistakes as it introduces mismatches between the backend/frontend and the database naming convention. Ideally, the database entry should match but I don't know if changing it there at this point is a good idea.  I agree with your suggestion.